neuro-cli/src/neuro_cli/formatters/config.py

```diff
@@ -100,6 +101,13 @@ def __call__(
         return RichGroup(*out)
 
 
+def _checkmark_character(value: bool) -> str:
+    if os.getenv("NEURO_EVENT_CATEGORY", "") == "WEB-CLI":
```

Comment:
I'm curious what is the value of `TERM` env var for web terminal?
Maybe we don't need a separate env?

Reply:
`TERM` is `xterm-256colors`, but Ivan Zubenko found a possible solution for the problem in the web-shell side. It's not only the term value, but also the locale. I'll test his solution and if it works, I'm happy to close this PR, as it's a band-aid